src/app/frontend/common/components/uploadfile/component.ts

```diff
@@ -13,14 +13,18 @@
 // limitations under the License.
 
 import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {MatDialog} from '@angular/material';
 import {HTMLInputEvent, KdFile} from '@api/frontendapi';
+import {AlertDialog, AlertDialogConfig} from 'common/dialogs/alert/dialog';
 
 @Component({
   selector: 'kd-upload-file',
   templateUrl: 'template.html',
   styleUrls: ['style.scss'],
 })
 export class UploadFileComponent {
+  constructor(private readonly matDialog_: MatDialog) {}
```

Comment:
Can you move it after class fields? After `filename` and before `onChange` method.